code/modules/surgery/__surgery_setup.dm

```diff
@@ -1,4 +1,4 @@
-#define  SURGERY_SKILLS_GENERIC				list(SKILL_ANATOMY = SKILL_ADEPT, SKILL_MEDICAL = SKILL_EXPERT)
-#define  SURGERY_SKILLS_DELICATE			list(SKILL_ANATOMY = SKILL_EXPERT, SKILL_MEDICAL = SKILL_EXPERT)
+#define  SURGERY_SKILLS_GENERIC				list(SKILL_MEDICAL = SKILL_EXPERT)
+#define  SURGERY_SKILLS_DELICATE			list(SKILL_MEDICAL = SKILL_EXPERT)
 #define  SURGERY_SKILLS_ROBOTIC				list(SKILL_DEVICES = SKILL_ADEPT)
-#define  SURGERY_SKILLS_ROBOTIC_ON_MEAT		list(SKILL_ANATOMY = SKILL_ADEPT, SKILL_DEVICES = SKILL_ADEPT) 
```

Comment:
Should replace it with SKILL_MEDICAL so every engineer isn't suddenly a surgeon too